### ISSUE TITLE
a11y checks are failing in CI

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#2009]
It looks like puppeteer has started to fail, see: https://github.com/dequelabs/cauldron/actions/runs/16122165498/job/45490421298
